tcases-core/src/main/java/org/cornutum/tcases/io/SystemTestHtmlWriter.java

```diff
@@ -13,13 +13,9 @@
 import org.apache.commons.collections4.IteratorUtils;
 import org.apache.commons.collections4.Predicate;
 import org.apache.commons.collections4.iterators.PeekingIterator;
-import org.apache.commons.io.IOUtils;
-import org.apache.commons.io.LineIterator;
 import org.apache.commons.lang3.StringUtils;
 
-import java.io.IOException;
-import java.io.OutputStream;
-import java.io.Writer;
+import java.io.*;
```

Comment:
Importing everything from a large JDK package can be problematic. I recommend importing each class actually used individually.

Reply:
fixed